betfair/betfair.py

```diff
@@ -90,6 +96,45 @@ def login(self, username, password):
             raise exceptions.BetfairLoginError(response, data)
         self.session_token = data['sessionToken']
 
+    def interactive_login(self, username, password):
```

Comment:
hi @jmcarp 

so having looked at this again, the issue I was having was that the 2 methods are not identical ... using ```curl``` returns 'SUCCESS', using ```requests``` returns FAIL;  

it may be because with ```requests```, *User-Agent* is set to ```python-requests/2.5.1 CPython/2.7.6 Darwin/14.3.0``` ?

any ideas? agree that your approach is better, but can't seem to make it work..!